src/main/kotlin/lotto/model/LottoGenerator.kt

```diff
@@ -0,0 +1,24 @@
+package lotto.model
+
+object LottoGenerator {
+    private const val LOTTO_PRICE = 1000
+
+    fun generateLottos(
+        paymentPrice: Int,
+        customLottos: Lottos,
+        lottoGeneratingStrategy: LottoGeneratingStrategy
+    ): Lottos {
```

Comment:
사실 LottoGenerator의 lottoGeneratingStrategy 는 운영환경에서 RandomLottoGenerating만을 사용하기 때문에 아래와 같이 default로 선언해주고 운영환경에서는 명시적으로 주입을 안해주는것이 나을것 같습니다~!

```suggestion
    fun generateLottos(
        paymentPrice: Int,
        customLottos: Lottos,
        lottoGeneratingStrategy: LottoGeneratingStrategy = RandomLottoGenerating
    ): Lottos {
```

그리고
generator가 아무런 프로퍼티 없이 정적 메서드만을 제공하는 함수로 사용되는 것이 나을지
아니면 생성될때 strategy를 주입받아서 생성하여 사용하는것이 좋을지 고민해보셔도 좋을것 같아요.